props/saber_fett263_buttons.h

```diff
@@ -4735,14 +4767,22 @@ SaberFett263Buttons() : PropBase() {}
           StopTrackPlayer();
           return true;
         }
-        ToggleBattleModeMultiBlast();
-        SaberBase::DoBlast();
+        if (CheckInteractiveBlast()) {
```

Comment:
This chunk seems to be repeated in a few places. I suggest a helper function

Reply:
added